sql/moz-fx-data-shared-prod/telemetry/active_users/view.sql

```diff
@@ -0,0 +1,193 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.telemetry.active_users`
+AS
+-- Distribution_id is only collected for Desktop and Fenix as of Q1/2024.
+-- See e.g. Firefox iOs https://dictionary.telemetry.mozilla.org/apps/firefox_ios?page=1&search=distribution_id
+-- TODO: Integrate distribution_id in fenix.firefox_android_clients & backfill to use it here.
+WITH fenix_distribution_id AS (
+  SELECT
+    client_info.client_id AS client_id,
+    sample_id AS sample_id,
+    DATE(submission_timestamp) AS submission_date,
+    ARRAY_AGG(
+      metrics.string.metrics_distribution_id IGNORE NULLS
+      ORDER BY
+        submission_timestamp ASC
+    )[SAFE_OFFSET(0)] AS distribution_id,
+  FROM
+    `moz-fx-data-shared-prod.fenix.baseline`
+  GROUP BY
+    client_id,
+    sample_id,
+    submission_date
+)
+-- Firefox Desktop
+SELECT
+  submission_date,
+  client_id,
+  CASE
+    WHEN isp_name = 'BrowserStack'
+      THEN CONCAT(app_name, ' ', isp_name)
+    WHEN distribution_id = 'MozillaOnline'
+      THEN CONCAT(app_name, ' ', distribution_id)
+    ELSE app_name
+  END AS app_name,
+  days_seen_bits,
+  days_active_bits,
+  IFNULL(mozfun.bits28.days_since_seen(days_active_bits) = 0, FALSE) AS is_dau,
+  IFNULL(mozfun.bits28.days_since_seen(days_active_bits) < 7, FALSE) AS is_wau,
+  IFNULL(mozfun.bits28.days_since_seen(days_active_bits), FALSE) AS is_mau,
```

Comment:
```suggestion
  IFNULL(mozfun.bits28.days_since_seen(days_active_bits) < 28, FALSE) AS is_mau,
```
The inequality is still needed here to return a bool; otherwise `IFNULL` throws an error:

```sql
SELECT IFNULL(mozfun.bits28.days_since_seen(4), FALSE)
# No matching signature for function IFNULL for argument types: INT64, BOOL. Supported signature: IFNULL(ANY, ANY)
```